twilight-model/src/poll/answer_count.rs

```diff
@@ -0,0 +1,43 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, Eq, Hash, PartialEq, Serialize)]
+pub struct AnswerCount {
+    /// The answer ID.
+    pub answer_id: u8,
```

Comment:
```suggestion
    pub id: u8,
```

This one is strange because `answer_id` is in the description.